pallet-tfgrid/src/lib.rs

```diff
@@ -30,6 +30,8 @@ pub type BalanceOf<T> =
 pub trait Config: system::Config + timestamp::Config {
     type Event: From<Event<Self>> + Into<<Self as frame_system::Config>::Event>;
     type Currency: Currency<Self::AccountId>;
+    /// Origin for sudo extrinsics
```

Comment:
sudo is just a pallet to dispatch calls as the Root origin, they are not sudo extrinsics